samples/cli_app.js

```diff
@@ -0,0 +1,50 @@
+const fs = require('fs');
```

Comment:
I would keep this for the sample ticket (API-188) to avoid any more scope creep. As it already got a bit out of hand.